### PR TITLE
Theme editor tweaks

### LINKED ISSUE #855: Theme dashboard UI cleanup
General cleanup of the theme dashboard area.

### PR DESCRIPTION
**Pull request by mikemurray**
resolves #855 

Tweaking editor styles
Renaming "theme" to "name" so we don't have instances of "theme.theme".

### CODE REVIEW

**Comment by aaronjudd** [comment]
getting an error: 
```
Exception from Tracker recompute function:
debug.js:41 ReferenceError: i18n is not defined
    at .<anonymous> (themeDetails.js:17)
    at view.js:199
    at Function.Template._withTemplateInstanceFunc (template.js:457)
    at view.js:197
    at Object.Blaze._withCurrentView (view.js:538)
    at viewAutorun (view.js:196)
    at Tracker.Computation._compute (tracker.js:323)
    at Tracker.Computation._recompute (tracker.js:342)
    at Object.Tracker._runFlush (tracker.js:481)
    at Object.Tracker.flush (tracker.js:441)
```

**Comment by aaronjudd** [comment]
:+1:

File: packages/reaction-i18n/private/data/i18n/en.json [hunk 1]
```diff
@@ -518,6 +518,16 @@
           "invitationSent": "Invitation Sent",
           "sendInvitation": "Send invitation"
         },
+        "reactionUI": {
+          "editTheme": "Edit Theme",
+          "duplicateTheme": "Duplicate Theme",
+          "duplicateThemeError": "Could't duplicate theme",
```
Comment (zenweasel) [inline packages/reaction-i18n/private/data/i18n/en.json]: Typo?

File: packages/reaction-ui/client/templates/dashboard/dashboard.js [hunk 2]
```diff
@@ -6,46 +6,48 @@ Template.uiDashboard.onCreated(function () {
...
+          onClick() {
+            Alerts.alert({
+              title: i18n.t("reactionUI.duplicateTheme", "Duplicate Theme"),
+              showCancelButton: true,
+              confirmButtonText: "Duplicate"
+            }, () => {
+              Meteor.call("ui/duplicateTheme", theme.name, (error) => {
+                if (error) {
+                  const alertDescription = i18next.t("reactionUI.duplicateThemeError", {
+                    defaultValue: "Could't duplicate theme"
```
Comment (zenweasel) [inline packages/reaction-ui/client/templates/dashboard/dashboard.js]: Same typo here

File: packages/reaction-ui/client/templates/themeEditor/themeEditor.js [hunk 3]
```diff
@@ -162,7 +162,7 @@ Template.uiThemeEditor.events({
 
     Meteor.call("ui/updateStyles", data, (error) => {
       if (error) {
-        Alerts.toast(`Could't update theme ${theme.theme}`);
+        Alerts.toast(`Could't update theme ${theme.name}`);
```
Comment (zenweasel) [inline packages/reaction-ui/client/templates/themeEditor/themeEditor.js]: And here

**Pull request merged**